libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/messages/reply/InReplyToView.kt

```diff
@@ -152,8 +153,10 @@ private fun ReplyToContentText(metadata: InReplyToMetadata?) {
     val text = when (metadata) {
         InReplyToMetadata.Redacted -> stringResource(id = CommonStrings.common_message_removed)
         InReplyToMetadata.UnableToDecrypt -> stringResource(id = CommonStrings.common_waiting_for_decryption_key)
-        is InReplyToMetadata.Text -> metadata.text
-        is InReplyToMetadata.Thumbnail -> metadata.text
+        // Add a limit to the text length to avoid a crash in Compose
+        is InReplyToMetadata.Text -> metadata.text.ellipsize(512)
+        // Add a limit to the text length to avoid a crash in Compose
+        is InReplyToMetadata.Thumbnail -> metadata.text.ellipsize(512)
```

Comment:
I remember that we already have a similar fix (limiting the size of text to avoid a Compose crash), but I can find it. Maybe I dreamt it?
Else maybe create a const for this `512` (not a big deal)?

Reply:
I had the same issue, I know it's there, I just don't know where 😅 .

Reply:
> I've tried unifying both in an extension function in a new commit.

That's also why I wanted to find where it was done before. Thanks for the update!